apps/meteor/tests/end-to-end/api/rooms.ts

```diff
@@ -3633,4 +3636,531 @@ describe('[Rooms]', () => {
 				});
 		});
 	});
+
+	describe('[/rooms.membersOrderedByRole]', () => {
+		const isEnterprise = Boolean(process.env.IS_EE);
+
+		let testChannel: IRoom;
+		let ownerUser: IUser;
+		let moderatorUser: IUser;
+		let memberUser1: IUser;
+		let memberUser2: IUser;
+		let customRole: IRole;
+
+		let ownerCredentials: { 'X-Auth-Token': string; 'X-User-Id': string };
+
+		before(async () => {
+			[ownerUser, moderatorUser, memberUser1, memberUser2] = await Promise.all([
+				createUser({ username: `a_${Random.id()}`, roles: ['admin'] }),
+				createUser({ username: `b_${Random.id()}` }),
+				createUser({ username: `c_${Random.id()}` }),
+				createUser({ username: `d_${Random.id()}` }),
+			]);
+
+			ownerCredentials = await login(ownerUser.username, password);
+
+			customRole = await createCustomRole({
```

Comment:
Should we check if enterprise here as well? I think it throws an error if not EE

Reply:
It should not throw error but resolve with undefined if not EE.